homeassistant/helpers/entity_component.py

```diff
@@ -154,6 +160,8 @@ def _async_setup_platform(self, platform_type, platform_config,
         except Exception:  # pylint: disable=broad-except
             self.logger.exception(
                 'Error while setting up platform %s', platform_type)
+        finally:
+            warn_task.cancel()
```

Comment:
The scheduling Task will execute in every case and we know what platform make trouble. In your case we never see a message...